apps/server/src/shared/infra/collaborative-storage/strategy/httpRequests/http-client.private.env.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dev": {
+    "CAPCAKESPASS": "4MbdOV3LSUOnolGzB2Ab8cL0tt"
```

Comment:
why is there a team specific thing here? :P

Reply:
Oh, you were careful, something was probably accidentally pushed :P